src/modules/apps/install.ts

```diff
@@ -63,8 +71,7 @@ export default {
 
     const app = optionalApp || `${manifest.vendor}.${manifest.name}@${manifest.version}`
     const apps = [app, ...options._.slice(ARGS_START_INDEX)].map(arg => arg.toString())
-
     log.debug('Installing app(s)', apps)
-    return installApps(apps)
+    return installApps(apps, options.r || options.registry)
```

Comment:
I think it should exist one `default` option to registries, currently we are using 'smartcheckout' as a default registry.

```js
return installApps(apps, options.r || options.registry || 'smartcheckout')
```